frontend/src/hooks/useEmoji.tsx

```diff
@@ -107,15 +120,7 @@ const useEmoji = () => {
     [getSenderName]
   );
 
-  useEffect(() => {
-    session?.on('signal', emojiHandler);
-
-    return () => {
-      session?.off('signal', emojiHandler);
-    };
-  }, [emojiHandler, session]);
```

Comment:
We handle this in the session context. Removing this silences the event listener console warning.